go.mod

```diff
@@ -9,43 +9,43 @@ require (
 	contrib.go.opencensus.io/exporter/stackdriver v0.8.0
 	fortio.org/fortio v1.3.1
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
-	github.com/fsnotify/fsnotify v1.6.0
```

Comment:
So not sure why the core go.mod and go.sum are being updated when it's just for an example? This probably should be reverted.

Reply:
I faced this issue `could not import sigs.k8s.io/controller-runtime/pkg/client (cannot find package "sigs.k8s.io/controller-runtime/pkg/client" in GOROOT or GOPATH)`. This issue was resolved after updating the go.mod file. Will revert and try again.👍🏻

Reply:
Yeah I don't know why the parent level go.mod needs to be updated? It's not even pushed to Cloud Build on submit.

Reply:
I faced errors when I tried building and pushing after reverting the changes in the root go.mod file and removing the vendor directory. Surprisingly, on the second attempt, I was able to push the image without these files, which is super confusing.

could not import sigs.k8s.io/controller-runtime/pkg/client (cannot find package "sigs.k8s.io/controller-runtime/pkg/client" in GOROOT or GOPATH)

cloud-build target error log - https://gist.github.com/Kalaiselvi84/a1bee237c519c6b1c9da2ebc66709f79